pkg/jailer/jailer.go

```diff
@@ -25,6 +28,14 @@ func CreateJail(name string) error {
 		return os.MkdirAll(path.Join(BaseJailPath, name), 0700)
 	}
 
+	if os.Getenv("MACHINE_REPO") != "rancher/machine" {
```

Comment:
probably should switch to a new setting and use CATTLE_MACHINE_REPO and get the setting value here so it's more standardized